tracer/README.MD

```diff
@@ -1,20 +1,117 @@
 ﻿# dd-trace-dotnet/tracer
 
-This repo ([dd-trace-dotnet](https://github.com/DataDog/dd-trace-dotnet)) contains the sources for the client-side components of the Datadog product suite for Application Telemetry Collection and Application Performance Monitoring for .NET Applications.
+This folder contains the source code for the Datadog .NET APM Tracer. The .NET Tracer automatically instruments supported libraries out-of-the-box and also supports custom instrumentation to instrument your own code.
 
-This directory (`/tracer`) contains the source for the __Tracer for .NET__ product, which is a part of that product suite.
+## Installation and usage
 
-### Copyright
+### Getting started
 
-Copyright (c) 2017 Datadog
-[https://www.datadoghq.com](https://www.datadoghq.com/)
+Configure the Datadog Agent for APM [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core#configure-the-datadog-agent-for-apm). For automatic instrumentation, install and enable the .NET Tracer [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core/?tab=windows#install-the-tracer).
 
-### License
+### Custom instrumentation
 
-See [license information](../LICENSE).
+The Datadog .NET APM Tracer allows you to manually instrument your application (in addition to automatic instrumentation). To use it, please follow [the NuGet package documentation](https://github.com/DataDog/dd-trace-dotnet/tree/master/docs/Datadog.Trace/README.md)
 
-## Note
+## Development
 
-Currently, this directory is empty.
-It is a placeholder for the Tracer sources, which will be available here in the future.
-In the meantime, the Tracer is available directly in the root of this repo.
+You can develop the tracer on various environments.
+
+### Windows
+
+#### Minimum requirements
+
+- [Visual Studio 2022 (v17)](https://visualstudio.microsoft.com/downloads/) or newer
+  - Workloads
+    - Desktop development with C++
+    - .NET desktop development
+    - .NET Core cross-platform development
```

Comment:
nit: .NET Core cross-platform development is no longer a workload in VS 2022. I'm not sure which workload places .NET 6 on the box though